src/direct-auth-idx/direct-auth-idx/Helpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNet.Identity;
+using Okta.Idx.Sdk;
+using System.Security.Claims;
+
+namespace direct_auth_idx
+{
+    public static class Helpers
+    {
+        public static ClaimsIdentity IdentityFromAuthResponse(string userName, AuthenticationResponse authnResponse) =>
```

Comment:
nit: Let's rename this method to GetIdentityFromAuthResponse (I'm guilty of using this name :) ).